app/src/main/java/ch/epfl/sweng/swenggolf/database/SignInActivity.java

```diff
@@ -66,20 +65,26 @@ private void initializeMAuthListener() {
         mAuthListener = new FirebaseAuth.AuthStateListener() {
             @Override
             public void onAuthStateChanged(@NonNull FirebaseAuth firebaseAuth) {
-                if (firebaseAuth.getCurrentUser() != null) {
-
-                    goToLogOut();
-
-                }
+            if (firebaseAuth.getCurrentUser() != null) {
+                goToWait();
+            }
             }
```

Comment:
The indentation is not correct. Please think about refactoring the code using AndroidStudio's tools (`Code > Reformat Code`).